src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -736,6 +988,15 @@ void cpu_convert(const void* srcPtr,
                   INTEL_CPU_CVT_FROM_BYTE_FP_LIST);
         if (!ctx.converted)
             OPENVINO_THROW("cpu_convert can't convert from: ", srcPrc, " precision to: ", dstPrc);
+#if defined(OPENVINO_ARCH_X86_64)
+    } else if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx) &&
```

Comment:
why jit impl is only for avx512_core_amx platform? Does PTL has avx512_core_amx? otherwise this implemented ConvertFP8Precision will not work on PTL.

Reply:
Thanks Chenhu for the comment! I've updated it to be avx512_core_fp16, which is the least isa requirement of these fp8 emitters https://github.com/openvinotoolkit/oneDNN/blob/c60a9946aa2386890e5c9f5587974facb7624227/src/cpu/x64/jit_avx512_core_fp8cvt.cpp#L621. For platforms that have inferior isa, it falls back to c++ impl of cpu plugin, which is still much faster than ngraph ref.

Reply:
avx512_core_fp16 is needed for f8/f16/f32 conversion. We need support bf16 as well, you use vcvtneps2bf16, not sure always available. May be jit_uni_vcvtneps2bf16 could be used here.

Reply:
Applied. Thanks Chenhu!